### PR TITLE
Create README-fr.md file

### PR DESCRIPTION
**Pull request by xroll**
# Add french translation README

**Description**
Add french translation README markdown

**Compatibility**
add one file README-fr.md in .github/

**Caveats**

### CODE REVIEW

File: .github/README-fr.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<div align="center">
+  <a href="https://nostalgic-css.github.io/NES.css/" target="_blank"><img src="https://user-images.githubusercontent.com/5305599/49061716-da649680-f254-11e8-9a89-d95a7407ec6a.png" alt="NES.css: NES-style  CSS framework" style="max-width: 100%;" width="600" height="315"></a>
+
+  <a href=".github/README-jp.md">日本語</a> / <a href=".github/README-zh-CN.md">简体中文</a> / <a href=".github/README-es.md">Español</a> / <a href=".github/README-pt-BR.md">Português</a> / <a href=".github/README-ru.md">Русский</a> / <a href=".github/README-fr.md" Français</a>
```
Comment (guastallaigor) [inline .github/README-fr.md:4]: Could you add the `<a href=".github/README-fr.md" Français</a>` in the other `README.md` files as well?

**Review by guastallaigor — CHANGES_REQUESTED** [review]
Thank you for your contribution 🎉 

Two things:
1. This PR has a commitlint error. 

Could you change your commit message to something like `feat: create readme-fr.md file` 🙇 

The git commands shoud be something like this:
```shell
# edit your commit message
$ git commit --amend -m "feat: create readme-fr.md file"

# force push branch
$ git push --force-with-lease origin develop
```

The second one I will add a comment below.
Also I can't review French, so I will take your word for it 😄

**Comment by xroll** [comment]
Understood, i will make changes and lint commit. Kind regards.

**Comment by guastallaigor** [comment]
ref #440

**Pull request closed without merging**